department-of-veterans-affairs/va.gov-cms
# VACMS-2347: Add drush jobs for deploy status and small deploy status /event refactor

## Description

See #2347. 

* Create new drush commands to interact with deploy mode
* Change deploy mode setting to save in drupal State rather than settings
* Small refactor of how `ContentExportPreTarEvent` works.

## Testing done

1. Run drush command `va-gov-get-deploy-mode` and see `DISABLED`
1. Visit CMS Export URL and get the download. Verify the timestamp is current.
1. Verify schema.json file is also created.
1. Wait 5 minutes.
1. Run drush command `va-gov-enable-deploy-mode`
1. Run drush command `va-gov-get-deploy-mode` and see `ENABLED`
1. Visit CMS Export URL and get the download. Verify the timestamp is 5 minutes old.
1. Run drush command `va-gov-disable-deploy-mode`
1. Run drush command `va-gov-get-deploy-mode` and see `DISABLED`


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/department-of-veterans-affairs/va.gov-cms/2605)
